zplugin-ros1/src/ros_to_zenoh_bridge/environment.rs

```diff
@@ -0,0 +1,82 @@
+//
+// Copyright (c) 2022 ZettaScale Technology
+//
+// This program and the accompanying materials are made available under the
+// terms of the Eclipse Public License 2.0 which is available at
+// http://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+// which is available at https://www.apache.org/licenses/LICENSE-2.0.
+//
+// SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+//
+// Contributors:
+//   ZettaScale Zenoh Team, <zenoh@zettascale.tech>
+//
+
+use std::str::FromStr;
+use rosrust::api::resolve::*;
+
+#[derive(Clone)]
+pub struct Entry<'a> {
+    pub name: &'a str,
+    pub default: String
+}
+impl<'a> Entry<'a> {
+    fn new<Tvar>(name: &'a str, default: Tvar) -> Entry<'a>
+    where
+        Tvar: ToString
+    {
+        return Entry{name, default: default.to_string()};
+    }
+
+    pub fn get<Tvar>(&self) -> Tvar
```

Comment:
Here I assume the whole goal is to retrive from `env` all the existing ROS1 configuration, right?

Reply:
I follow the approach to store all the config in `env` and directly set and get it when needed. `Environment` struct creates `Entry` objects wrapped around exact config options - and these `Entry` options provide all necessary interface to work with config option.